src/style.css

```diff
@@ -1 +1,68 @@
 /* Styles go here */
+body {
+  margin: 0;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav__list {
+  align-items: center;
+  display: flex;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__list > :not(:first-child) {
+  margin-left: 20px;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+.nav__link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+  white-space: nowrap;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+  cursor: pointer;
+}
+
+.nav__item {
+  list-style: none;
+}
+
+.nav__item .is-active::after {
+  content: "";
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height: 4px;
+  border-radius: 8px;
+  background-color: #00acdc;
+  border-color: #00acdc;
+}
```

Comment:
```suggestion
.is-active::after {
  content: "";
  position: absolute;
  bottom: 0;
  left: 0;
  right: 0;
  height: 4px;
  border-radius: 8px;
  background-color: #00acdc;
  border-color: #00acdc;
}
```

Reply:
Also would be nice to set styles to class at first and then to pseudoelements (or pseudoclasses)